pom.xml

```diff
@@ -65,14 +65,15 @@
     </dependency>
     <!-- https://mvnrepository.com/artifact/io.vertx/vertx-web-templ-jade -->
     <dependency>
-        <groupId>io.vertx</groupId>
-        <artifactId>vertx-web-templ-jade</artifactId>
+      <groupId>io.vertx</groupId>
+      <artifactId>vertx-web-templ-pug</artifactId>
+      <version>5.0.4</version>
```

Comment:
also here you specify the 5.0.4 , so the other vertx.x deps should be it as well. Perhaps introduce a property so you only specify once?